pkg/debuginfo/upload.go

```diff
@@ -55,7 +54,7 @@ func (u *Uploader) UploadAll(ctx context.Context, srcDbgInfo map[SourceInfo]io.R
 	var result *multierror.Error
 	for src, r := range srcDbgInfo {
 		if err := u.Upload(ctx, src, r); err != nil {
-			level.Warn(u.logger).Log(
+			level.Debug(u.logger).Log(
```

Comment:
Should we keep it as a warning? If uploading fails, something has gone pretty wrong, no?

Reply:
There's an error returned at the end that actually gets logged. We don't need additional warning if we aren't actually debugging.